src/pages/public_registration/action_accordions/DuplicateWarningModal.tsx

```diff
@@ -0,0 +1,92 @@
+import OpenInNewOutlinedIcon from '@mui/icons-material/OpenInNewOutlined';
+import { Box, Button, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { Link } from 'react-router-dom';
+import { Modal } from '../../../components/Modal';
+import { dataTestId } from '../../../utils/dataTestIds';
+import { getRegistrationLandingPagePath } from '../../../utils/urlPaths';
+
+interface DuplicateWarningModalProps {
+  isOpen: boolean;
+  toggleModal: () => void;
+  onConfirmNotDuplicate: () => void;
+  duplicateId: string | undefined;
+}
+
+export const DuplicateWarningModal = ({
```

Comment:
Etter å ha tenkt meg litt om så virker det som denne komponenten har samme formål som `ConfirmDialog.tsx`. Kunne innholdet vi har her blitt sendt som children til den i stedet kanskje? 🤔 

Gevinsten med det er å ha en fast måte å håndtere tilfeller hvor bruker skal bekrefte eller avbryte en handling. ConfirmDialog er en dum dialog som bare viser content basert på hva som kommer fra parent, og tillater bruker å bekrefte eller avbryte det brukeren prøver på

Reply:
Jeg kan godt bytte ut <`Modal`> med <`ConfirmDialog`>, men jeg syns fortsatt vi bør ha en `DuplicateWarningModal` slik at vi slipper å ha all den view-koden inne i `PublishingAccordion` og da er spørsmålet hvor mye vi tjener på å bruke det ene istedet for de andre siden de begge innkapsler en del av modal-logikken.

Reply:
Da vil den se slik ut: 
![image](https://github.com/BIBSYSDEV/NVA-Frontend/assets/1130244/941db47b-2e2d-4fce-a949-6f395cd15527)